inference-engine/tests/ngraph_functions/src/broadcast.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+std::shared_ptr<ngraph::Node> makeBroadcast(const ngraph::Output<Node> &in,
```

Comment:
If I remember right we already have make_broadcast function in the ngraph::builder static library. Why didn't you use this function?

Reply:
@ilyachur that's right, but `builder::opset1::make_broadcast` produces only `Broadcast::v1` in `EXPLICIT` mode.

Reply:
Yeah, but why we cannot extend `make_broadcast` function? 
Also opset1 and opset5 are the same for Broadcast layer. We can bump the opset version in the broadcast builder.

Reply:
@ilyachur Broadcast:v3 was introduced in opset3 (with new `bidirectional` mode).
What's more `builder::opset1::make_broadcast` uses inverted logic of axes for explicit mode - take axes broadcasted in output (Broadcast:v0 logic), where Broadcast:v1,v2 take axes from input which will be broadcasted.
But you are right it should be refactored - I think that after removing legacy opset0